api/android/api/src/main/jni/nnstreamer-native-api.c

```diff
@@ -24,20 +24,24 @@
 #include "nnstreamer-native.h"
 
 /* nnstreamer plugins and sub-plugins declaration */
+#if !defined (NNS_SINGLE_ONLY)
 GST_PLUGIN_STATIC_DECLARE (nnstreamer);
 GST_PLUGIN_STATIC_DECLARE (amcsrc);
 extern void init_filter_cpp (void);
 extern void init_filter_custom (void);
 extern void init_filter_custom_easy (void);
 extern void init_filter_tflite (void);
-#if defined (ENABLE_SNAP)
-extern void init_filter_snap (void);
-#endif
 extern void init_dv (void);
 extern void init_bb (void);
 extern void init_il (void);
 extern void init_pose (void);
 extern void init_is (void);
+#endif /* NNS_SINGLE_ONLY */
```

Comment:
I think `/* NNS_SINGLE_ONLY */` could lead some confusion because it was derived from `#if !defined (NNS_SINGLE_ONLY)`

Reply:
do you want to change macro to '#ifndef'? most headers use it.

Reply:
I meant the comment. When I see `#endif /* NNS_SINGLE_ONLY */`, it looks like a pair of `#if defined (NNS_SINGLE_ONLY)`, not `#if !defined (NNS_SINGLE_ONLY)`.
I am not sure which comment is better, but I think the current comments make some confusion.

Reply:
ok, I will remove all.